code/src/test/kotlin/mocks/TokenResponses.kt

```diff
@@ -0,0 +1,21 @@
+package mocks
+
+import com.google.api.client.auth.oauth2.TokenResponse
+
+
+// Our expiration threshold is 10 seconds
+val successfulTokenResponse = TokenResponse().apply {
+    setScope("example")
+    setRefreshToken("refresh_token")
+    setAccessToken("access_token")
+    setExpiresInSeconds(3600)
+    setTokenType("token_type")
+}
+
+val aboutToExpireSuccessfulTokenResponse = successfulTokenResponse.clone().apply {
+    setExpiresInSeconds(9)
+}
+
+val oneSecondForAboutToExpireSuccessfulTokenResponse = successfulTokenResponse.clone().apply {
+    setExpiresInSeconds(11)
+}
```

Comment:
I don't understand these. Are the just named oddly?

Reply:
I followed the naming convention in older versions of the `sdk-core` module, however yes they sound odd without a context.

The context is, a token can be in one of 3 states:
- **Expired**: cannot be used, we need to refresh.
- **About to Expire**: Token is still valid but would expire within 10 seconds or less, we refresh early.
- **Ideal State**: Token is fine and has more than 10 seconds till it's about to expire.

Here we try to mock these states.

Reply:
The code doesn't seem to reflect the "expired" state based on your description. And the "ideal" state based on your description isn't named as such.

They seem odd to me because they all look like they are "about to expire" since 9, 10, and 11 seconds are pretty much the same.